docs/getting-started/advanced-guide/schema.md

```diff
@@ -0,0 +1,430 @@
+# Schemas
+This page delves into more detail on Schemas. As seen in the other guides, there are two main components to schemas: Elements and Types.
+
+Schemas are often separated into two JSON files (`elements.json` & `types.json`), but it's worth remembering that your schema can be broken up further if required.
+
+For example if you have a Graph that is made up of 5 different data sources, you may find it easier to develop and maintain by splitting your schema into multiple parts, for example 5 Element Schemas and 1 Types schema.
+Alternatively, a single overall schema file could be used for Elements and Types. There's also no requirement to have a single Types schemas.
+When you construct your Gaffer graph you must provide all the Schema parts. If provided separately, these will then be merged together to form an internal schema used for the Graph.
+
+When using Java directly, you don't need to use JSON files to store you schemas - although this is still recommended. See the [Java API](#java-api) section for more about interacting with Schemas in Java.
+
+The sections below walkthrough the features of Schemas in detail and explain how to accurately model your data in Gaffer.
+
+
+## Elements schema
+The Elements schema is designed to be a high level document describing what information your Graph contains, i.e. the different kinds of edges and entities and the list of properties associated with each.
+Essentially this part of the schema should just be a list of all the entities and edges in the graph. 
+Edges describe the relationship between a source vertex and a destination vertex. 
+Entities describe a vertex. We use the term "element" to mean either and edge or and entity.
```

Comment:
```suggestion
Entities describe a vertex. Edges describe the relationship between a source vertex and a destination vertex.
We use the term "element" to mean either an edge or an entity.
```

I would swap these just so it flows a little better and also just fixed the grammar.